Release 83.0.0 (#3018)
MetaMask/snaps: packages/examples/packages/cronjobs/CHANGELOG.md

## [Unreleased]

## [3.0.0]

### Added

- **BREAKING:** Add new method to cronjob example snap to test durations ([#3016](https://github.com/MetaMask/snaps/pull/3016))

>   - One of the existing RPC methods was renamed as part of this change.

Reviewer: ```suggestion
  - The `scheduleNotification` RPC method was renamed to `scheduleNotificationWithDate`.
```
